orchagent/aclorch.cpp

```diff
@@ -59,14 +60,19 @@ acl_rule_attr_lookup_t aclMatchLookup =
     { MATCH_INNER_L4_DST_PORT, SAI_ACL_ENTRY_ATTR_FIELD_INNER_L4_DST_PORT }
 };
 
-acl_rule_attr_lookup_t aclL3ActionLookup =
+static acl_rule_attr_lookup_t aclL3ActionLookup =
 {
-    { PACKET_ACTION_FORWARD,                    SAI_ACL_ENTRY_ATTR_ACTION_PACKET_ACTION },
-    { PACKET_ACTION_DROP,                       SAI_ACL_ENTRY_ATTR_ACTION_PACKET_ACTION },
-    { PACKET_ACTION_REDIRECT,                   SAI_ACL_ENTRY_ATTR_ACTION_REDIRECT }
+    { ACTION_PACKET_ACTION,                    SAI_ACL_ENTRY_ATTR_ACTION_PACKET_ACTION },
+    { ACTION_REDIRECT_ACTION,                  SAI_ACL_ENTRY_ATTR_ACTION_REDIRECT },
```

Comment:
My preference is to keep these orch data structures clean and aligned with SAI types and backwards compatibility handled as exceptional cases in the code.

If I will add PACKET_ACTION_DROP/FORWARD to this structure it will mess with new entries.
Despite it will remove exceptional cases to handle backward compatibility in existing flow to program rules, since this map is used to fill in capabilities table, this will add more exceptional cases in this new code and in future when we cleanup deprecated items in this map we will also need to cleanup exceptional cases in the code in any case.

Reply:
agreee